app/_layout.tsx

```diff
@@ -47,6 +48,8 @@ export default function RootLayout() {
     PublicSans_700Bold,
   });
 
+  useDrizzleStudio(expo);
```

Comment:
should this be wrapped in a check for non-production env? e.g. `process.env.NODE_ENV !== "production"`